setup.cfg

```
@@ -4,15 +4,15 @@ universal=1
 [metadata]
 name = panedr
 author = Jonathan Barnoud
-author-email = jonathan@barnoud.net 
+author_email = jonathan@barnoud.net
 summary = Read and manipulate Gromacs energy files
 license = LGPL
-description-file =
+description_file =
     README.rst
 long_description_content_type = text/x-rst
-home-page = https://github.com/jbarnoud/panedr
-requires-python = '>=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*'
-classifier = 
+home_page = https://github.com/jbarnoud/panedr
+python_requires = >=2.7,!=3.0.*,!=3.1.*,!=3.2.*,!=3.3.*,!=3.4.*
```

```suggestion
python_requires = >=3.8
```

Unless @jbarnoud disagrees, we should just make our lives easier here and start with NEP29 (purely on the basis that it makes packaging so much easier). We could be a bit more lax and make this 3.7+ but support is rapidly being dropped.